lib/logthrsource/tests/test_logthrsourcedrv.c

```diff
@@ -198,8 +222,6 @@ Test(logthrsourcedrv, test_threaded_source_suspend)
   s->num_of_messages_to_generate = 5;
```

Comment:
optional: Here we have an implicit `blocking_post = FALSE` , set by the `g_new0` in the `test_threaded_source_driver_init_method` function. It would be harder to misuse this test if we have an explicit function pointer.